schemas/2017-03-01/Microsoft.ApiManagement.json

```diff
@@ -67,6 +67,17 @@
           ],
           "description": "SKU properties of the API Management service."
         },
+        "identity": {
```

Comment:
>identity [](start = 9, length = 8)

do you want to update the tests in /tests/2017-03-01/ to include these changes?

Reply:
@TianoMS  added test